src/api/route-services/experiment.js

```diff
@@ -265,6 +248,26 @@ class ExperimentService {
     return cellSetData;
   }
 
+  async patchCellSets(experimentId, patch) {
+    const cellSetsObject = await this.getCellSets(experimentId);
+    const { cellSets: cellSetsList } = cellSetsObject;
+
+    /**
+     * The $remove operation will replace the element in the array with an
+     * undefined value. We will therefore remove this from the array.
+     */
```

Comment:
which operation in particular does this `$remove` operation in the comment refer to? Maybe the comment is outdated?

Reply:
The `$remove` operator refers to an operator by the JSON patch library. You can see that we use it in the worker when we send a PATCH during cell set updates.

Reply:
I see, then might be more clear if you modify the comment to `The $remove operation we use in the worker will ...`?

Reply:
fixed